src/index.js

```diff
@@ -1,25 +1,36 @@
 // import _ from 'lodash';
 import './style.css';
-import enterImg from './img/enter.png';
-import trashImg from './img/trash.png';
-import task from './modules/taskList.mjs';
-import listen from './modules/addtask.mjs';
-import newLi from './modules/taskBuild.mjs';
 
 const todoList = document.getElementById('todoList');
 
-// Enter Button
-const addBtn = document.getElementById('enter');
-const enter = new Image();
-enter.src = enterImg;
-enter.style.width = '100%';
-addBtn.appendChild(enter);
+const list = [
```

Comment:
- [ ] **Required:** @banzaman Kindly check to be sure you submitted your intended work. 

This is what I can see at the moment.
![image](https://user-images.githubusercontent.com/27709832/161402430-961a80fa-2b55-4333-9477-00457f28fdbd.png)

- Most files in the branch appear to be deleted.

- [ ]  Add an event listener on the checkbox which should update on the localStorage upon user interaction.
- [ ] Implement a "Clear all completed" function.